src/services/wallet.service.js

```diff
@@ -84,7 +88,24 @@ export async function createWallet(network, name, mnemonic, passphrase) {
     //account public
     const accountPub = await cmd(getPublicCmd(accountPrv.stdout));
     if(accountPub.stderr) throw accountPub.stderr;
-    
+
+    //payment priv/pub keys (needed to get verification keys)
+    const paymentPrv = await cmd(getChildCmd(accountPrv.stdout, "0/0"));
+    if(paymentPrv.stderr) throw paymentPrv.stderr;
+    const paymentPub = await cmd(getChildCmd(accountPub.stdout, "0/0"));
+    if(paymentPub.stderr) throw paymentPub.stderr;
+    //payment signiing key (needed to get verification keys)
+    const paymentSKeyPath = path.resolve(walletDir, paymentSigningKeyFile);
+    const paymentSigningKey = getBufferHexFromFile(paymentPrv.stdout).slice(0, 128) + getBufferHexFromFile(paymentPub.stdout);
+    const paymentSKey = `{
+        "type": "PaymentExtendedSigningKeyShelley_ed25519_bip32",
+        "description": "Payment Signing Key",
+        "cborHex": "5880${paymentSigningKey}"
+    }`
+    // public [extended] verification key
+    const extendedVerificationKeyPath = path.resolve(walletDir, extendedVerificationKeyFile);
+    const verificationKeyPath = path.resolve(walletDir, verificationKeyFile);
```

Comment:
are any of these keys sensitive?

Reply:
`paymenSkey`, that's why we cleanup [here.](https://github.com/nothingalike/lift-wallet/pull/9/files#diff-76115ce7d41e2ae056afea681e93a327959d67151b960e6fe2e76da1aa7b62f1R154)